website/src/writing-php-code/phpdoc-types.md

```diff
@@ -102,7 +102,23 @@ function repair(string $type, int $wheels): void
 }
 ```
 
-Additionally `value-of<BackedEnum>` is supported.
+Additionally `value-of<BackedEnum>` is supported:
+
+```php
+enum Suit: string
+{
+    case Hearts = 'H';
+    case Spades = 'S';
+}
+
+/**
+ * @param array<value-of<Suit>, int> $count
+ */
+function foo(array $count): void
+{
+    // Example for $count: ['H' => 2, 'S' => 3]
```

Comment:
In reality it's going to be `array<'H'|'S', int>`: https://phpstan.org/r/ba72dae8-a092-4570-a81e-0a5a0646c15d